shell/common/skia_event_tracer_impl.cc

```diff
@@ -50,7 +49,11 @@ inline T BitCast(const U& u) {
   memcpy(&t, &u, sizeof(t));
   return t;
 }
-#endif
+
+union SkiaTraceValueUnion {
```

Comment:
AFAICT this isn't actually required, right?  You're only using this for the bool value type right now, which you should be able to just (more simply and without having to copy types) convert with a "!!" (i.e. "!!p_arg_values[i]").

Reply:
"!!" may not meet expectation. I think the reason is that 'TraceValueUnion' is 64 bits, and it is not initialized to 0. Testing on the iOS platform，"!!" always get 'true' value.